service/provider/src/main/java/gov/va/vro/service/provider/services/HealthAssessmentErrCheckProcessor.java

```diff
@@ -0,0 +1,28 @@
+package gov.va.vro.service.provider.services;
+
+import gov.va.vro.model.HealthDataAssessment;
+import gov.va.vro.service.provider.ExternalCallException;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class HealthAssessmentErrCheckProcessor implements Processor {
+  @Override
+  @SneakyThrows
+  public void process(Exchange exchange) {
+
+    HealthDataAssessment hda = exchange.getMessage().getBody(HealthDataAssessment.class);
+
+    if (hda.getErrorMessage() != null) {
+      log.info("Health Data Assessment sent back with error : {}", hda.getErrorMessage());
```

Comment:
```suggestion
      log.warn("Health Data Assessment sent back with error : {}", hda.getErrorMessage());
```
If this log should be investigated then use `log.error`.

Reply:
@yoomlam  SVC-lighthouse is already doing a log.error. If you want the main vro app docker to also error it to ensure it gets seen, that's fine. or it can be warn to not dupe up on messages. No preference on my end. but i am curious as to yours.

Reply:
talked with yoom. we'll do error here.